### PR TITLE
Fixes #18190: Add agent-server troubleshooting section

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/18190

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: src/reference/modules/administration/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -1,5 +1,75 @@
...
+  `/var/log/rudder/webapp/TODAY.stderrout.log` for lines about your node's inventory:
+
+** If you have an *Invalid signature* message
+
+*** It could be because of a change of node id or agent key. Compare the certificate fingerprint and node id from the web interface and the `rudder agent info` output.
+*** There was a problem with the signature process on the node. Particularly, when there is a problem with "openssl" command, the signature may be empty. Check the content of signature file for that inventory in `/var/rudder/inventories/failed/xxx-yyy-zzz.ocs.sig-DATE`
+*** The node sending the inventory is not the same as the one registered in Rudder. This can show a security problem, but it may also be due to two nodes having the same nodeID for some reason (a VM cloned followed with an update of the agent key of one the cloned node, but no change of the node ID).
+In this case `rudder agent factory-reset` should be run after VM clone.
+
+** If you have an *Invalid inventory* message this is likely a bug, please open an issue with
```
Comment (ncharles) [inline src/reference/modules/administration/pages/troubleshooting.adoc:27]: you already have it on l10

File: src/reference/modules/administration/pages/troubleshooting.adoc [hunk 2]
```diff
@@ -1,5 +1,75 @@
...
+   the provided error message.
+** If you have no message related to your node
```
Comment (ncharles) [inline src/reference/modules/administration/pages/troubleshooting.adoc:29]: i don't understand this sentence in this context

File: src/reference/modules/administration/pages/troubleshooting.adoc [hunk 3]
```diff
@@ -1,5 +1,75 @@
...
+
+* If it gives an *Unspecified server refusal* error:
+
+** Check if the node has actually been accepted on the server and that a first policy generation finished after acceptation
+** Check the policy server logs on the server or relay for more details
+
+* If it succeeds then the problem is probably in the reporting process. Run `rudder agent run -i` and
+  check the messages about reporting at the end:
+
+** If it fails with a curl error:
```
Comment (ncharles) [inline src/reference/modules/administration/pages/troubleshooting.adoc:50]: i'm pretty sure update doesn't curl, does it ?

Comment (ncharles) [reply src/reference/modules/administration/pages/troubleshooting.adoc:50]: maybe you meant rudder agent run -ui ?

**Review by ncharles — CHANGES_REQUESTED** [review]
some remarks about the text

File: src/reference/modules/administration/pages/troubleshooting.adoc (hunk 2 above)
Comment (amousset) [reply src/reference/modules/administration/pages/troubleshooting.adoc:29]: we're still on the sublevel of "check webapp logs for messages related to your node's id" item, I'll make it more precise

File: src/reference/modules/administration/pages/troubleshooting.adoc (hunk 3 above)
Comment (amousset) [reply src/reference/modules/administration/pages/troubleshooting.adoc:50]: it's under " check the messages about reporting at the end"

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**